serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/quarkus-dev-ui-extension/quarkus-dev-ui-monitoring-page.adoc

```diff
@@ -0,0 +1,27 @@
+= Monitoring in Kogito Serverless Workflow Tools extension
+:compat-mode!:
+// Metadata:
+:description: Monitoring in Kogito Serverless Workflow Tools extension
+:keywords: kogito, workflow, serverless, Quarkus, Dev UI, monitoring
+
+In the Kogito Serverless Workflow Tools extension, the *Monitoring* page displays default dashboards that might be useful
```

Comment:
```suggestion
In the {product_name} Tools extension, the *Monitoring* page displays default dashboards that might be useful
```
Can you use this variable instead? https://github.com/kiegroup/kogito-docs/blob/main/serverlessworkflow/antora.yml#L13